docs/spec/resolution.md

```diff
@@ -6,19 +6,19 @@
 
 1. Upon invocation of resolution, retrieve all observed operations for the [DID Unique Suffix](#did-unique-suffix) of the DID URI being resolved.
 2. If record of the DID being published has been observed, proceed to Step 3. If there is no observed record of the DID being published, skip all remaining [Operation Compilation](#operation-compilation) steps and process the DID as follows:
-    1. If the DID URI ****does not**** include a `initial-state` _DID URL Parameter_ with a Method-specific prefix that matches the name of the implementation, abort resolution and return _Unresolvable_.
-    2. If the DID URI ****does**** include a `initial-state` _DID URL Parameter_ with a Method-specific prefix that matches the name of the implementation, process the DID as follows:
-        1. Split the `initial-state` _DID URL Parameter_ string value by the period (`.`) character, and ensure the resulting array contains ****exactly**** two (2) members. If the resulting array contains fewer than two members, abort resolution and return _Unresolvable_.
+    1. If the DID URI is a [_Long-Form DID URI_](#long-form-did), abort resolution and return _Unresolvable_.
```

Comment:
you mean short form?

Reply:
Probably, I will take a look at the context

Reply:
Yep, this was a paste error - fixed in added commit.